src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -102,7 +102,7 @@
 		// Map container formats to their "audio variant".
 		mutableAudioOrVideoContainers.put(FormatConfiguration.MP4, new AudioVariantInfo(new M4A(), FormatConfiguration.M4A));
 		mutableAudioOrVideoContainers.put(FormatConfiguration.MKV, new AudioVariantInfo(new MKA(), FormatConfiguration.MKA));
-		mutableAudioOrVideoContainers.put(FormatConfiguration.OGG, new AudioVariantInfo(new OGG(), FormatConfiguration.OGA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.OGG, new AudioVariantInfo(new OGA(), FormatConfiguration.OGA));
```

Comment:
@Nadahar `f:ogg` was not changed by `f:oga` in the audio support section of the renderers.